clang/lib/AST/ASTContext.cpp

```diff
@@ -13462,13 +13462,114 @@ static ElaboratedTypeKeyword getCommonTypeKeyword(const T *X, const T *Y) {
                                             : ElaboratedTypeKeyword::None;
 }
 
+static NestedNameSpecifier *getCommonNNS(ASTContext &Ctx,
+                                         NestedNameSpecifier *X,
```

Comment:
X and Y are getting confusing as names when reviewing this :/  I know it isn't LHS and RHS, but even something that shows a bit more context (FirstNNS, SecondNNS, NNS1, NNS2, etc) would be helpful for reviewing/reading.